packages/util/package.json

```diff
@@ -3,6 +3,7 @@
   "version": "0.12.1",
   "description": "browser utils for @ffmpeg/*",
   "main": "./dist/cjs/index.js",
```

Comment:
With type set to `module`, these file extensions will either need to be `.cjs` or have a package.json file created in their respective ./cjs folder that has its type set to `commonjs`. Otherwise the module resolution API will treat them all as ESM.

Reply:
It is quiet hard to find a perfect solution for building and bundling, will test and see if tsup is better in this scenario if I have time. Thanks!